apps/builder/app/shared/tree-utils.ts

```diff
@@ -34,6 +34,16 @@ export const getAncestorInstanceSelector = (
   return instanceSelector.slice(ancestorIndex);
 };
 
+export const areInstanceSelectorsEqual = (
+  left?: InstanceSelector,
+  right?: InstanceSelector
+) => {
+  if (left === undefined || right === undefined) {
+    return false;
+  }
+  return left.join(",") === right.join(",");
```

Comment:
fun way to do the same `String(left) === String(right)` but also wondering if this is going to be a very frequently called function and actual walking over array would be faster

Reply:
I think join is used in .toString() of arrays. So kinda same, join is just more explicit.